Tests/Test/ResponseTest.swift

```diff
@@ -1,14 +1,17 @@
 import XCTest
+import Routes
 import Server
 
 class ResponseTest: XCTestCase {
     
     let parser = Parser()
-    let response = Response()
+    static let routes: [String: Route] = ["/": Root(), "/hello": Hello(), "/tea": Tea(), "/coffee": Coffee(), "/parameters": Parameters(), "/cookie": Cookie(), "/eat_cookie": EatCookie(), "/redirect": Redirect(), "/form": Form(), "/method_options": MethodOptions(), "/method_options2": MethodOptions2()]
```

Comment:
Seems like you may run into some issues keeping these routes up to date

Reply:
Yes, I definitely "felt the pain" on that one. I plan on addressing that so I don't have to write these routes twice. Once for test and once for Main.